### DISCUSSION TITLE
--listen functionality for Spaces

### DISCUSSION THREAD

**Discussion by Animation-IDEMI** [lllyasviel/stable-diffusion-webui-forge#1767] [Q&A]
Need --listen functionality for Spaces to host on local server and access from any PC on LAN.

Currently WebUI is accessible on local IP 10.0.0.10:7860 from any PC on LAN , but Spaces only launch on localhost:7861 and are not accessible from 10.0.0.10:7861 from PCs on LAN. Is there any existing solution?

![1](https://github.com/user-attachments/assets/495470b3-0c4d-40e6-9eb7-ff548022b5ee)

**Comment by kmdtukl** [comment]
It works as it is. The site says, with --listen key active: "Currently Running: http://localhost:7861/", but look at the log in the console - it says: "Running on local URL: http://0.0.0.0:7861".

**Reply by Animation-IDEMI** [reply]
@kmdtukl Somehow it wasn't working yesterday but it is working now. Thanks anyway.
